internal/acceptance/openstack/objectstorage/v1/objects_test.go

```diff
@@ -135,7 +136,7 @@ func TestObjects(t *testing.T) {
 
 	// Copy the contents of one object to another.
 	copyOpts := objects.CopyOpts{
-		Destination: cName + "/" + oNames[1],
+		Destination: "/" + cName + "/" + oNames[1],
```

Comment:
As per [the docs](https://docs.openstack.org/api-ref/object-store/#copy-object): the Copy `Destination` header must be in the form of `/container/object`.